pom.xml

```diff
@@ -2023,6 +2023,46 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>com.github.ekryd.sortpom</groupId>
+                    <artifactId>sortpom-maven-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <executions>
+                        <execution>
+                            <id>sortpom-warn</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Warn</verifyFail>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>sortpom-fail</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Stop</verifyFail>
+                            </configuration>
+                        </execution>
+                    </executions>
+                    <configuration>
```

Comment:
Let's add `<spaceBeforeCloseEmptyElement>true</spaceBeforeCloseEmptyElement>` since that matches our style and matches the Maven documentation

Reply:
Also add `<sortModules>true</sortModules>` which should be same as the existing order

Reply:
Applied both.